examples/crossmodel.py

```diff
@@ -48,6 +49,10 @@
         print('Consuming offer')
         await consuming_model.consume("admin/test-cmr-1.mysql")
 
+        print('Exporting bundle')
```

Comment:
might move this log into the with statement so you can log where the file/path is expected to be in case folks are using this log to chase anything down? I guess it's just an example...never mind